### DISCUSSION TITLE
How to work with pac outputs programmatically

### DISCUSSION THREAD

**Discussion by martin-dreyer-teck** [microsoft/powerplatform-vscode#344] [Q&A] [answered]
When running a command such as `pac admin list` from PowerShell the output is an array of strings.

![image](https://user-images.githubusercontent.com/59576723/200395404-f1c256af-3371-4dcf-b222-fb5d38aed02a.png)

This output does not appear to be a structure that we can address programmatically, because there's some info text in the first row and because each row is simply seen as a single string. with no reliable way to figure out column values.

Am I missing something?

**Comment by devkeydet** [comment, marked as answer]
try

`pac admin list --json`
